geode-protobuf/src/main/java/org/apache/geode/protocol/protobuf/utilities/ProtobufUtilities.java

```diff
@@ -140,7 +141,7 @@
 
   /**
    * This creates a MessageHeader
-   *
+   * 
```

Comment:
what's up with adding all this white trailing white space? Is this from spotless?